.claude/skills/translate-batch/scripts/repair_manifest.py

```
@@ -0,0 +1,90 @@
+    m = json.loads(MANIFEST.read_text())
+    e = m["entries"] if "entries" in m and isinstance(m.get("entries"), dict) else m
+    wrap = "entries" in m and isinstance(m.get("entries"), dict)
+    added = 0
+    for cp in sorted(CONTENT_ROOT.rglob("*.md")):
+        rel = cp.relative_to(ROOT).as_posix()
+        if rel in e:
+            continue
+        fm = read_fm(cp)
+        if not fm.get("upstream_sha") or not fm.get("translated_at"):
+            continue
+        # 1) フロントマターの upstream_path があれば優先（移動/改名済みページ対応）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)